docker-compose.yml

```diff
@@ -18,8 +18,17 @@ services:
 
   cvat_redis:
     container_name: cvat_redis
-    image: redis:7.0-alpine
+    image: eqalpha/keydb:x86_64_v6.3.2
     restart: always
+    command: [
+      "keydb-server",
+      "/etc/keydb/keydb.conf",
+      "--storage-provider", "flash", "/data/flash",
+      "--maxmemory", "5G",
+      "--maxmemory-policy", "allkeys-lfu",
```

Comment:
We're using the same KeyDB instance for both asynchronous tasks and caching. For cached objects, this policy makes sense, but wouldn't it cause problems for task queues? We could start randomly losing background tasks if their DB entries get evicted.

Reply:
https://docs.keydb.dev/docs/flash:

> With KeyDB on FLASH you can set maxmemory to the amount of DRAM you want allocated without the fear of losing > data when exceeded. Instead the data is removed from DRAM but remains in FLASH where it is not lost and can still be > served.

Reply:
Ah, alright. That's reassuring.